### DISCUSSION TITLE
Any plans for the Windows 10 Search box?

### DISCUSSION THREAD

**Discussion by LforGithub** [valinet/ExplorerPatcher#1914] [Ideas]
This is a great restoration project but the old windows 10 search box would be SO much appreciated

**Comment by milestprower92** [comment]
> This is a great restoration project but the old windows 10 search box would be SO much appreciated

No longer possible in 22H2 and 23H2 because Microsoft has removed the code of the old search box from the old taskbar and there are no plans to have it added back due to the fact that the search icon does the same thing as the search box.
